### ISSUE TITLE
Add commands to the stryker cli

### ISSUE THREAD

**Issue by nicojs** [stryker-mutator/stryker#171]
I think we should add commands to the stryker CLI. For example this call:

```bash
stryker -c stryker.conf.js
```

Would become:

```bash
stryker run -c stryker.conf.js
```

This way we can later add more commands. For example:

```bash
stryker diagnose -c stryker.conf.js
```

This could for example diagnose to see if your test suite is stateless and can be ran in any order (needed for coverage analysis to work).

An other example could be:

```bash
stryker debug -c stryker.conf.js
```

This could load the olinitial test run in the same node process (no isolation) for debug purposes. 

This would be a breaking change, so we could include this in the 0.5.0 release.

@simondel what do you think?

**Comment by simondel** [comment]
Good idea! But I would still support the current `stryker -c stryker.conf.js` command and just pass it to `stryker run`.

**Comment by nicojs** [comment]
Well... i don't like that.

1. First of all: its quite common to expect the help text when executing a cli that expects commands. People might get confused. Also, running stryker takes a long time. Doing it by accident would piss people of.
2. Second: i would like to use the command cli's way of parsing a command: https://www.npmjs.com/package/commander#specify-the-argument-syntax (i know i shouldn't let technical details cloud my judgement, but i don't want to complicate stuff too much ;)).
3. The last point is simple: how would people even know there is a `debug`, `statistics` or `whatever` command if they always run the cli without a command. Its not intuitive to expect commands to be there if you can run them without it.

**Comment by nicojs** [comment]
Closed with #181
